### DISCUSSION TITLE
Tie 'skin selection" to 'profile selection"

### DISCUSSION THREAD

**Discussion by ogami1972** [Drewol/unnamed-sdvx-clone#588] [Q&A] [answered]
I am using the experimental gear skin, and it would be great if the three players on this machine have their own crew and appeal cards.

I currently address this by having players go to settings and select a skin that matches their name, which is a copy of the skin folder with personalized player name, crew, etc.

It would be great to streamline this by making the skin change occur when we change profiles by pressing the two FX buttons on the controller. The players don't currently use the offset features, and this might encourage that as well.

The name on the high scores still changes when the player name changes, but we can live with that, lol.

**Comment by ogami1972** [comment, marked as answer]
Hello!

I found,, through tinkering, that this actually is already done, just needed a "skin=" in the profile .cfg file.

**Reply by Jaybberwocky** [reply]
Hey Ogami1972 - I'm looking to create a similar build at home and you seem to have done exactly what I would love to do (personalised the skin, and have highscores save locally). Is there any chance you can share with me what you did to achieve all of this?
